packages/frontend/src/components/accounts/create/initial_deposit/InitialDepositWrapper.js

```diff
@@ -63,24 +64,45 @@ export function InitialDepositWrapper({ history }) {
 
     const checkFundingAddressBalance = async () => {
         if (fundingNeeded) {
-            const account = wallet.getAccountBasic(implicitAccountId);
-            const state = await account.state();
-
-            if (new BN(state.amount).gte(new BN(MIN_BALANCE_TO_CREATE))) {
-                setFundingNeeded(false);
-                setInitialDeposit(state.amount);
-            }
+            await Mixpanel.withTracking("CA Check balance from implicit",
+                async () => {
+                    try {
+                        const account = wallet.getAccountBasic(implicitAccountId);
+                        const state = await account.state();
+                        if (new BN(state.amount).gte(new BN(MIN_BALANCE_TO_CREATE))) {
+                            Mixpanel.track("CA Check balance from implicit: sufficient");
+                            setFundingNeeded(false);
+                            setInitialDeposit(state.amount);
+                            return;
+                        } else {
+                            console.log('Insufficient funding amount');
+                            Mixpanel.track("CA Check balance from implicit: insufficient");
+                        }
+                    } catch(e) {
+                        if (e.message.includes('does not exist while viewing')) {
+                            return;
+                        }
+                        throw e;
+                    }
+                },
+                (e) => {
+                    throw e;
+                }
```

Comment:
```suggestion
                }
```
re-throwing from the errorOperation with no other logic is equivalent to just not defining one at all :)